### PR TITLE
docs: improve the tab panel text with interactive content

### PR DESCRIPTION
**Pull request by spraguejdev**
Closes #https://github.com/carbon-design-system/carbon/issues/11291

This pr accomplished two items. 

1. Consistency for interactive content for tab 2 in the storybook. 
2. Keeps the tab label text and panel text, DRY, by using an enum for the text. 🙂 

#### Changelog

**New**

- {{new thing}}

**Changed**

- {{change thing}}

**Removed**

- {{removed thing}}

#### Testing / Reviewing

1. Navigate to any of the react tab stories http://localhost:3000/?path=/story/components-tabs--contained
2. Verify the text for the tab label  for each story 
4. Verify the text for the tab panel for each story 
5. Enjoy 🍿

### CODE REVIEW

**Comment by spraguejdev** [comment]
I have read the DCO document and I hereby sign the DCO.

File: packages/react/src/components/Tabs/next/Tabs.stories.js [hunk 1]
```diff
@@ -106,32 +158,52 @@ export const IconOnly = (args) => (
...
 
 export const Contained = (args) => (
   <Tabs>
     <TabList aria-label="List of tabs" contained {...args}>
-      <Tab>Tab Label 1</Tab>
-      <Tab>Tab Label 2</Tab>
-      <Tab disabled>Tab Label 3</Tab>
-      <Tab title="Tab Label 4 with a very long long title">
-        Tab Label 4 with a very long long title
+      <Tab>{tabPanelEnums.tab1.linkText}</Tab>
```
Comment (tay1orjones) [inline packages/react/src/components/Tabs/next/Tabs.stories.js:180]: The first tab has no text in the deploy preview

image

Maybe it's just this?
```suggestion
      <Tab>{tabPanelEnums.tab1.labelText}</Tab>
```

File: packages/react/src/components/Tabs/next/Tabs.stories.js [hunk 2]
```diff
@@ -29,44 +29,87 @@ export default {
   },
 };
 
+const tabPanelEnums = {
```
Comment (tay1orjones) [inline packages/react/src/components/Tabs/next/Tabs.stories.js:32]: Small nit, but I wouldn't be opposed to a shorter/simpler name here

```suggestion
const tabs = {
```

Or if you'd like, could even break these out into their own variables to cut down on the chaining by one level
```
const tab1 = { labelText: '', panelText: '' };
const tab2 = { ... };
...etc
```

**Review by tay1orjones — CHANGES_REQUESTED** [review]
Super cool of you to just pick this up out of the blue, thanks! Just a couple comments.

**Comment by abbeyhrt** [comment]
Hey @spraguejdev! Super cool for you to jump and do this! I love the consistency in the second tab panel. One of the goals of our stories is to have them be as copy/paste-able as possible and using an enum for the text like this makes it harder for each story to be easily copied and pasted.  It definitely looks very repetitive in that file, but in this, that is on purpose.  With the way that storybook breaks down each story, the "story" panel on storybook is the copy/paste-able snippet that we are looking for.

**Comment by spraguejdev** [comment]
> Hey @spraguejdev! Super cool for you to jump and do this! I love the consistency in the second tab panel. One of the goals of our stories is to have them be as copy/paste-able as possible and using an enum for the text like this makes it harder for each story to be easily copied and pasted. It definitely looks very repetitive in that file, but in this case, that is on purpose. With the way that storybook breaks down each story, the "story" panel on storybook is the copy/paste-able snippet that we are looking for.

Hey @abbeyhrt! Thanks for the review and for allowing me to be a noob 🙂 to how yall do stuff. I'm here to learn and I totally understand what you are recommending. I'll be making this change asap. 

Thanks and nice to virtually work with you 🥇

**Comment by aledavila** [comment]
@[spraguejdev](https://github.com/spraguejdev) just wondering if you had a chance to look at the comments or if you still wanted to work on it. Thanks

**Comment by aledavila** [comment]
Closing. Fixes have already been made

**Pull request closed without merging**